pyproject.toml

```diff
@@ -97,7 +98,7 @@ docs = [
 dev = [
     # Tensorflow
     # cf. https://github.com/mindee/doctr/pull/1461
-    "tensorflow>=2.15.0,<3.0.0",
+    "tensorflow[and-cuda]>=2.15.0,<3.0.0",
```

Comment:
@odulcy-mindee we should do the same here as above :)

Reply:
Oh yeah :+1: